jax/_src/pallas/mosaic/lowering.py

```diff
@@ -2267,6 +2267,47 @@ def _shift_right_logical_lowering_rules(ctx: LoweringRuleContext, x, d):
 skip_mlir_conversions.add(lax.shift_right_logical_p)
 
 
+# based on https://github.com/openxla/xla/blob/a7a09d56c3599123f8148bbf3e44c9ebc04624b9/xla/mlir_hlo/mhlo/transforms/chlo_legalize_to_hlo/chlo_legalize_to_hlo.cc#L644-L802
+def _erf_inv_32_helper(x):
+  k_degree = 9
+  w_less_than_5_constants = [
```

Comment:
How about `w_lt_5_constants` and `w_ge_5_constants`?

Reply:
Sounds good. Done